docs/guides/ip-allowlist.mdx

```diff
@@ -0,0 +1,21 @@
+---
+id: allowlist
+title: Configure Ory Network IP addresses as allow list
+sidebar_label: Ory Network IP Addresses
+---
+
+To enhance your security measures while using the Ory Network, we've narrowed down the source IP addresses for all of our egress
+traffic (webhook, emails etc.) to specific IPs.
+
+These addresses will be used in the EU region:
+
+- 34.22.170.75
+- 35.242.228.133
+
+These addresses will be used in the US region:
+
+- 35.245.69.134
+- 34.94.111.83
+
+By setting your firewall rules to allow traffic only from these IP addresses, you'll be adding an extra layer of protection
+against any unwarranted access attempts.
```

Comment:
Should we add a note about how we would update the IPs? So send out a notice on the security newsletter, have 1 month of notice time, ...

Reply:
Yes, we do update customers beforehand like we did last year, however, here it is not a place to describe any "change notifications" as they apply to different areas as well.